.rubocop.yml

```diff
@@ -6,45 +6,61 @@ require:
 
 inherit_from: .rubocop_todo.yml
 
+#### Local Settings
+
 AllCops:
   Exclude:
     - db/**/*
     - node_modules/**/*
     - bin/**/*
     - vendor/**/*
 
+Lint/MissingSuper: # (new in 0.89)
+  Enabled: true
+  Exclude:
+    - app/components/**/*
+
 Metrics/BlockLength:
   Exclude:
     - spec/**/*
 
+Metrics/ClassLength:
+  Max: 150
+
+Metrics/MethodLength:
+  Max: 20
```

Comment:
cops actually changed in this PR

Reply:
For me, 20 pushes the limits of readability. I'm ok with it here, as we've discussed, but I'd prefer we avoid making it our "standard".

Reply:
@aaron-collier make me an offer.

Reply:
I'd be happy at 15.

Reply:
Just added a commit with method length max at 15.  It restores two inline exceptions for method length.  I can live with it ... unless we get a lot more of those for methods that have no clear refactor.

Reply:
I wouldn't die on this hill but... I'll ask: how would folks feel about 10?

Reply:
10 is too small for me.  it is the default;  this is me wanting to increase the default.

Reply:
or Mike, are you saying you would prefer to keep it at 10, but you could compromise?  While I am saying I would prefer it to be 20, and I have compromised down to 15 ... but we are not agreeing yet?

Reply:
@ndushay I thought the default was 5, not 10. :laughing: let's stick with 15 for now.